python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -505,6 +624,37 @@ def start_sso_renewal(self):
         self._sso_countdown_timer.start()
         self._session_renewal_active = True
 
+    def _on_cookie_added(self, cookie):
+        """
+        In PySid2/Qt5, we have limited access to query/update the CookieStore. We
+        must rely on signals called at each cookie addition and removal. We thus
+        keep a cache of the present CookieStore state in our own cookie jar.
+        This class existed in PySide/Qt4, but then was used as the actual cookie
+        store.
+        """
+        # This logging is commented out due to its verbosity.
```

Comment:
We've sometimes resorted to having an env var that adds extra logging, this way you don't need to uncomment it and also it can be provided to clients to debug even further if the need ever arises.

Reply:
Fixed. Used an environment variable to set an instance variable (._developer_mode)